regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_strict_mode.groovy

```diff
@@ -30,7 +30,7 @@ suite("test_partial_update_strict_mode", "p0") {
                 `last_access_time` datetime NULL 
             ) ENGINE = OLAP UNIQUE KEY(`id`) 
             COMMENT 'OLAP' DISTRIBUTED BY HASH(`id`) 
-            BUCKETS AUTO PROPERTIES ( 
+            BUCKETS 4 PROPERTIES ( 
```

Comment:
Why we need to change auto bucket to static bucket? It looks this pr have no connected with auto bucket?

Reply:
just to ensure that the table has more than one buckets to check if the calculation is correct if the table has more than one bucket.